README.rst

```diff
@@ -41,120 +41,13 @@ PyAnsys Geometry is a Python client library for the Ansys Geometry service.
 Usage
 -----
 
-There are two different ways of getting started with the Geometry service and its client-library, PyAnsys Geometry.
+There are different ways of getting started with using the Geometry service and its client-library, PyAnsys Geometry.
 
-Using PyAnsys Geometry launcher
-^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+For more information, please refer to the
+`Getting Started <https://geometry.docs.pyansys.com/version/stable/getting_started/index.html>`_ documentation.
```

Comment:
```suggestion
For more information, see
`Getting Started <https://geometry.docs.pyansys.com/version/stable/getting_started/index.html>`_ documentation.
```
We always want to use this format: For X, see Y. Wouldn't we want to insert a target and use the :ref" directive here?

Reply:
We can't use the ``:ref:``  directive because this README file is outside of the documentation. 😢